core/src/main/java/org/apache/iceberg/rest/RESTSessionCatalog.java

```diff
@@ -393,6 +438,15 @@ private LoadTableResponse loadInternal(
 
   @Override
   public Table loadTable(SessionContext context, TableIdentifier identifier) {
+    if (!endpoints.contains(ResourcePaths.V1_LOAD_TABLE)) {
+      throw new NoSuchViewException(
+          "Unable to load view %s.%s: Server does not support endpoint %s %s",
```

Comment:
Why state that this is because the endpoint isn't supported? I think that's clear from where the exception originated and it isn't useful for people that read the exception message.

Reply:
Stack traces are no always available... not in all call contexts. Sometimes (often?) only the exception message is reported.

Reply:
@rdblue this feels like a narrow case (I'm not sure you can support views without being able to load tables.  Technically there are things you could do, but they're largely not meaningful).  However, I think we need to be explicit that the failure was due to `load_table` not being supported.  Otherwise, it's unclear as to why it wasn't able to load because:  The message says unable to load, the exception says it doesn't exist, but the truth is that it's not a supported operation.

Reply:
I'm okay with moving forward with it.

Reply:
I've updated this back to `Unable to load view %s.%s: Server does not support endpoint %s`